### DISCUSSION TITLE
Request to remember the position of the desktop icons for each resolution separately

### DISCUSSION THREAD

**Discussion by yaaaaa** [linuxdeepin/developer-center#3366] [Features Request & Ideas | 特性请求 & 头脑风暴]
Request to remember the position of the desktop icons for each resolution separately

**Comment by BLumia** [comment]
I think it was the old behavior and got removed during development. Could you specify the use case? Thanks!

**Comment by yaaaaa** [comment]
When I change the screen resolution from 1920x1080 to 1024x768, then immediately put it back the way it was originally, the icons do not return to their place.
